data/experiment-recipe-samples/desktop-95-multifeature.json

```diff
@@ -0,0 +1,75 @@
+{
+  "appId": "firefox-desktop",
+  "appName": "firefox_desktop",
+  "application": "firefox-desktop",
+  "arguments": {},
+  "branches": [
+    {
+      "feature": {
+        "featureId": "unused-feature-id-for-legacy-support",
+        "enabled": false,
+        "value": {}
+      },
+      "features": [
+        {
+          "enabled": true,
+          "featureId": "pocketNewtab",
+          "value": {}
+        },
+        {
+          "featureId": "upgradeDialog",
+          "value": {
+            "enabled": false
+          }
+        }
+      ],
+      "ratio": 1,
+      "slug": "control"
+    },
+    {
+      "features": [
+        {
+          "enabled": true,
+          "featureId": "pocketNewtab",
+          "value": {
+            "compactLayout": true,
+            "lastCardMessageEnabled": true,
+            "loadMore": true,
+            "newFooterSection": true
+          }
+        },
+        {
+          "featureId": "upgradeDialog",
+          "value": {
+            "enabled": true
+          }
+        }
+      ],
+      "ratio": 1,
+      "slug": "treatment"
+    }
+  ],
+  "bucketConfig": {
+    "count": 10000,
+    "namespace": "firefox-desktop-multifeature-test",
+    "randomizationUnit": "normandy_id",
+    "start": 0,
+    "total": 10000
+  },
+  "channel": "nightly",
+  "endDate": null,
+  "featureIds": ["upgradeDialog", "pocketNewtab"],
```

Comment:
The `feature` (the deprecated one) featureId is `"unused-feature-id-for-legacy-support"`, I would expect it to match one of the `Id`s from the `features` array (or be listed here in `featureIds` as well).

Reply:
Ah, `"unused-feature-id-for-legacy-support"` is placeholder we always use to prevent errors not a real-world value :+1: nevermind the parent comment then.